README.md

```diff
@@ -0,0 +1,72 @@
+# webauthn
+
+Implementation of strong authentication with the webauthn standard and FIDO2.
+Strong authentication is an authentication method using a physical key.
+
+For a more thorough introduction see these two nice articles:
+
+- [introduction](https://medium.com/@herrjemand/introduction-to-webauthn-api-5fd1fb46c285)
+- [verifying fido2 responses](https://medium.com/@herrjemand/verifying-fido2-responses-4691288c8770)
+
+## Installation
+
+```js
+npm install webauthn
+```
+
+## usage
+
+Webauthn is composed of two parts @webauthn/client and @webauthn/server
+
+### On the browser
+
+```js
+import { 
+    registrationChallengeToPublicKey, 
+    loginChallengeToPublicKey, 
+    publicKeyToJSON 
+} from '@webauthn/client';
+```
+
+- registrationChallengeToPublicKey:
```

Comment:
For each function (server + client), can you describe the API of them?
What is a challenge, what is a public key?

Reply:
I will change the whole api in the next PR